apollo-router-core/src/services/router_service.rs

```diff
@@ -258,10 +259,14 @@ impl PluggableRouterServiceBuilder {
         let (execution_service, execution_worker) = Buffer::pair(
             ServiceBuilder::new().service(
```

Comment:
Suggest move the layer so that it gets executed before plugins:
 
`ServiceBuilder::new().layer(ForbidHttpGetMutations::default()).service(`